src/compiler/gcc.rs

```diff
@@ -275,6 +283,21 @@ where
                     OsString::from(arg.flag_str().expect("Compilation flag expected"));
             }
             Some(ProfileGenerate) => profile_generate = true,
+            Some(ProfileUse(path)) => {
+                if compiler_kind != CCompilerKind::Clang {
```

Comment:
Another way you could do this, without adding the compiler kind, is to simply add -fprofile-use to the clang list of arguments (see `args::test_multi_search`), in which case it would probably be more appropriate to call this `ClangProfileUse` rather than `ProfileUse`.

Reply:
Ah, you can "override" arguments that way? That does sound like the better approach for this case.